pkg/machines/store.es6

```diff
@@ -17,15 +17,14 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { createStore, applyMiddleware } from 'redux';
+import { createStore, applyMiddleware, compose } from 'redux';
 import reducer from './reducers.es6';
 import thunkMiddleware from 'redux-thunk';
 
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
To clarify for the uninitiated: In production this would normally be undefined, but if you enable the devtools extensions in the browser they would "hook in" here?

There wasn't much justification or explanation in #9472, nor do I know much about this, but it looks reasonable to me.

Reply:
So to clarify, in #9472 I removed the custom implementation of the middleware and replaced with existing solultion by redux-thunk (Redux Thunk middleware allows you to write action creators that return a function instead of an action). By mistake I removed the support for redux extension for chrome devtools because I thought it was part of the middleware for async actions having no experience with redux at all.
So this PR is not reverting #9472, it's just bringing back support for the redux extension.

To answer your question, in production, when the extension is not installed, we’re using Redux compose. But if it's installed you can use it, yes.

Note: I am just following the DOCS from here https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup, if there is some redux expert here I would gladly get some input if this is the correct way.

Reply:
Makes sense, thanks!